README.md

```diff
@@ -2,8 +2,28 @@
 
 This experiment comes in two phases:
 
-  1. Generate Blackbox data
-  2. Run the experiments
+  1. Install dependencies
+  2. Generate Blackbox data
+  3. Run the experiments
+
+
+## Install dependencies
+
+To run the experiments you need to install Python 2.7 and Rust. There are
+several ways to do install Rust, but [rustup](https://rustup.rs/) is generally
```

Comment:
several ways to ~~do~~ install Rust

Reply:
Fixed in https://github.com/softdevteam/error_recovery_experiment/pull/27/commits/19467c1fcde635bfeaa8286bd2e976bb860f59c5